src/databases/functions/fcmToken/findAllTokenByUserId.js

```diff
@@ -0,0 +1,23 @@
+// eslint-disable-next-line no-unused-vars
+const {Model} = require('sequelize');
+const {sequelize} = require('../../models');
+
+/**
+ *
+ * @param {Model} fcmTokenModel
+ * @param {String} userId
+ */
+module.exports = async (fcmTokenModel, userId) => {
+  const userTargetTokens = sequelize.query(
+    'SELECT DISTINCT(token) as token from user_token WHERE user_id = :userId',
+    {
+      replacements: {
+        userId
+      },
+      raw: true,
+      type: sequelize.QueryTypes.SELECT
+    }
+  );
+
+  return userTargetTokens;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `findAllTokenByUserId` is using raw SQL query to fetch the user tokens. While this isn't necessarily a problem, it's not taking full advantage of Sequelize's ORM capabilities. Using Sequelize's built-in methods can help prevent SQL injection attacks and improve code readability.

Here's how you could refactor this function to use Sequelize's `findAll` method:

```diff
-  const userTargetTokens = sequelize.query(
-    'SELECT DISTINCT(token) as token from user_token WHERE user_id = :userId',
-    {
-      replacements: {
-        userId
-      },
-      raw: true,
-      type: sequelize.QueryTypes.SELECT
-    }
-  );
+  const userTargetTokens = await fcmTokenModel.findAll({
+    where: {
+      user_id: userId
+    },
+    attributes: ['token'],
+    distinct: true
+  });

   return userTargetTokens;
```
This way, you're leveraging Sequelize's built-in protections and making your code more maintainable and readable.


<!-- This is an auto-generated comment by OSS CodeRabbit -->